linters/jshint.py

```diff
@@ -0,0 +1,13 @@
+def options(ctx):
+    ctx.add_option('--jshint', action='store_true', help="Run JSHint on the JS files in the build")
+
+def configure(ctx):
+    if ctx.options.jshint:
+        try:
+            ctx.find_program('jshint', var='JSHINT')
+        except ctx.errors.ConfigurationError:
+            print "jshint was not found"
+
+def build(ctx):
+    if ctx.env.JSHINT:
+        ctx(rule='${JSHINT} ${SRC}', source=ctx.path.ant_glob('src/**/*.js'), name='jshint')
```

Comment:
Would it be worth implementing a feature here to demonstrate how this is done, or do you think it's worth keeping the simplicity in this example?